docs-gen/includes/quickstart/Dockerfile

```diff
@@ -1,10 +1,27 @@
-# TFX build
-FROM tensorflow/tfx:1.13.0
+FROM python:3.10.12 as builder
 
 RUN mkdir /data
 RUN wget https://raw.githubusercontent.com/tensorflow/tfx/master/tfx/examples/penguin/data/labelled/penguins_processed.csv -P /data
 
+RUN pip install poetry==1.7.1
+
+COPY pyproject.toml .
+COPY poetry.lock .
+
+RUN poetry config virtualenvs.create true
+RUN poetry config virtualenvs.in-project true
+RUN poetry install
```

Comment:
can we use `--without dev` as suggested in the article?

Reply:
I suppose we don't do it here because we have no dev dependencies 😅

Reply:
added `--no-root` in e437d31dec58b158038a13043857f7b673f935cb